src/Bot.Discord/Components/InviteButtons.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Color_Chan.Discord.Commands.MessageBuilders;
+using Color_Chan.Discord.Core.Common.API.DataModels;
+using Color_Chan.Discord.Core.Common.Models;
+
+namespace Bot.Discord.Components;
+
+public static class InviteButtons
```

Comment:
This isn't a component. So you can put this directly in the `InviteCommands` class.